db/migrate/20190602130719_rename_etablissement.rb

```diff
@@ -0,0 +1,9 @@
+class RenameEtablissement < ActiveRecord::Migration[5.0]
+  def self.up
+    rename_table :etablissements, :etablissements_v2
+  end
+
+  def self.down
+    rename_table :etablissements_v2, :etablissements
+  end
```

Comment:
Par curiosité : je sais que ActiveRecord peut induire tout seul les migrations inverses en cas de rollback, il sait pas le faire pour un renommage de table ?!

Reply:
apparement non c'est la syntaxe recommandée. Je vais quand même faire un essai avec la syntaxe normale pour voir

Reply:
Je crois que les up and down c'est plus trop d'actualité depuis environ deux ans, je suis etonné que ce soit le cas ici.

Pour le renommage de colonne en tout cas il gère : https://github.com/etalab/sirene_as_api/blob/master/db/migrate/20170925105117_rename_statut_diffusion_to_prospection.rb

Reply:
done